app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -130,20 +130,11 @@ fun ParkingDetailsScreen(
                                     if (userSignedIn.value) {
                                       if (isFavorite) {
                                         userViewModel.removeFavoriteParkingFromSelectedUser(
-                                            selectedParking.uid)
-                                        Toast.makeText(
-                                                context,
-                                                "Removed From Favorites!",
-                                                Toast.LENGTH_SHORT)
-                                            .show()
+                                            selectedParking)
```

Comment:
Why were the toasts removed?

Reply:
The toasts are here to provide feedback when the UI is not enough. However, here the heart filling and turning red is enough for the user to understand the parking was successfully added to their favorites